packages/support/resources/views/components/modal/index.blade.php

```diff
@@ -94,11 +103,23 @@ class="fixed inset-0 z-40 flex items-center min-h-screen p-4 overflow-y-auto tra
                     'max-w-5xl' => $width === '5xl',
                     'max-w-6xl' => $width === '6xl',
                     'max-w-7xl' => $width === '7xl',
+                    'fixed inset-4 w-auto' => $isFullscreen,
                 ])
             >
                 @if ($header)
-                    <div class="filament-modal-header px-4 py-2">
-                        {{ $header }}
+                    <div class="filament-modal-header px-4 py-2 flex items-center justify-between">
+                        <div>
+                            {{ $header }}
+                        </div>
+
+                        @if ($isFullscreen)
+                            <x-filament-support::icon-button
```

Comment:
Maybe we could introduce the close button for all modals instead of just the fullscreen ones?

Reply:
I'm ok with having in on all modals.

Reply:
Fixed in latest commit.